Source/Core/DolphinQt/Debugger/CodeDiffDialog.h

```diff
@@ -13,6 +13,13 @@ class QLabel;
 class QPushButton;
 class QTableWidget;
 
+enum UpdateType
+{
+  Include,
+  Exclude,
+  Backup
+};

```

Comment:
UpdateType is only used internally by CodeDiffDialog, so you can move the declaration inside the private section of the declaration to get it out of global scope.

It can also be an enum class.